keyboards/zeix/singa/kohaku/keyboard.json

```
@@ -0,0 +1,717 @@
+{
+    "manufacturer": "SINGA",
+    "keyboard_name": "Kohaku",
+    "maintainer": "itsme-zeix",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true
+    },
+    "indicators": {
+        "caps_lock": "GP29"
+    },
+    "matrix_pins": {
+        "cols": ["GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1", "GP0"],
+        "rows": ["GP27", "GP28", "GP10", "GP11", "GP18", "GP19", "GP23", "GP24", "GP25", "GP26"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "brightness_steps": 8,
+        "led_count": 28,
+        "saturation_steps": 8,
+        "sleep": true
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x8888",
+        "vid": "0x4C27"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layout_aliases": {
+        "LAYOUT_65_ansi": "LAYOUT_ansi",
+        "LAYOUT_65_ansi_split_bs": "LAYOUT_ansi_split_bs",
+        "LAYOUT_65_ansi_tsangan": "LAYOUT_ansi_tsangan",
+        "LAYOUT_65_ansi_tsangan_split_bs": "LAYOUT_ansi_tsangan_split_bs",
+        "LAYOUT_65_iso_split_bs": "LAYOUT_iso_split_bs",
+        "LAYOUT_65_iso_tsangan": "LAYOUT_iso_tsangan",
+        "LAYOUT_65_iso_tsangan_split_bs": "LAYOUT_iso_tsangan_split_bs",
+        "LAYOYT_65_iso": "LAYOUT_iso"
```

- aliases removed, to reflect Community Layout support
- Community Layout support added
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs", "65_iso_blocker", "65_iso_blocker_split_bs", "65_iso_blocker_tsangan", "65_iso_blocker_tsangan_split_bs"],
```